src/Mod/Sketcher/Gui/ViewProviderSketch.h

```diff
@@ -635,9 +643,6 @@ class SketcherGuiExport ViewProviderSketch : public PartGui::ViewProvider2DObjec
     /// snap points x,y (mouse coordinates) onto grid if enabled
     void snapToGrid(double &x, double &y);
 
-    /// moves a selected constraint
-    void moveConstraint(int constNum, const Base::Vector2d &toPos);

```

Comment:
Ha! Trying to bypass the checkpoint? ;) 

If this is for DrawHandler, you need to use the attorney.